api/resources/translations/messages-en.properties

```
@@ -421,6 +421,8 @@ close = Close
 configuration.authorization = Roles & permissions
 configuration.date.format.help = Moment.js format
 configuration.enable.token.login = Enable login via SMS link
+configuration.enable.sso.login = Enable login Via SSO
+configuration.enable.sso.login.help = If this is enabled, the user will be redirected to the SSO login page when they click "Login with SSO" button.
```

suggestion: this wording is going to be a bit tricky. We need to try to find the balance of providing enough info while not getting so in-depth that things are more confusing.  I like that we are explicit about the "Login with SSO" button. I think we also need to make it clear that the provided username value must match (we can come back and update this later if/when we add more logic to the matching process.)

What do you think about:

```suggestion
configuration.enable.sso.login.help = When logging in, the user must select the "Login with SSO" button. Requires the user name provided here to match the user's unique identifier from the external SSO provider.
```